src/dns/aardvark.rs

```diff
@@ -116,16 +116,35 @@ impl Aardvark {
         // After https://github.com/containers/aardvark-dns/pull/148 this command
         // will block till aardvark-dns's parent process returns back and let
         // aardvark inherit all the fds.
-        Command::new(aardvark_args[0])
+        let out = Command::new(aardvark_args[0])
             .args(&aardvark_args[1..])
-            .stdin(Stdio::inherit())
-            .stdout(Stdio::inherit())
-            .stderr(Stdio::inherit())
+            .stdin(Stdio::null())
```

Comment:
So... Stdio::null() forces it to use a local buffer? That's rather unintuitive

Reply:
no, it does not it pus /dev/null on it, we obviously don't want to send anything on stdin so this is the right thing, we also do not care about stdout so there as well.

output() causes the default to be `Stdio::piped()` (write to a buffer) so that is why i didn't set stderr here

Reply:
Should we not clear Stdout then, and print it if there's an error? Not that I expect there to be much there, but it might be good policy.

Reply:
sorry I don't understand your question, clear what exactly?

stdin -> dev/null
stdout -> dev/null (aardvark-dns never writes there)
sdterr -> buffered (when aardvark exits with error read all and return message in error string)

Reply:
Basically saying we should buffer stdout in case we decide to write to it in the future. But I won't argue too loudly since we've not actually using it now.

Reply:
I will complain loudly if I ever see someone printing on stdout in aardvark-dns :)